src/h3lib/lib/algos.c

```diff
@@ -749,7 +749,10 @@ H3Error H3_EXPORT(gridRingUnsafe)(H3Index origin, int k, H3Index *out) {
  * @return 0 (E_SUCCESS) on success.
  */
 H3Error H3_EXPORT(maxPolygonToCellsSize)(const GeoPolygon *geoPolygon, int res,
-                                         int64_t *out) {
+                                         uint32_t flags, int64_t *out) {
+    if (flags != 0) {
+        return E_FAILED;
```

Comment:
Worth adding `E_UNSUPPORTED_FLAGS` or similar?

Reply:
I'm fine either way. I don't believe the specific error code is considered part of the API contract so I don't see an issue with it changing.